native/acter/src/api/client/sync.rs

```diff
@@ -568,6 +568,28 @@ impl Client {
                 if !changed_rooms.is_empty() {
                     trace!(?changed_rooms, "changed rooms");
                     me.refresh_rooms(changed_rooms).await;
+
+                    // we have seen changes, see if we have
```

Comment:
account data within rooms _didn't_ yet trigger proper notifications to the UI. This is doing exactly that now.